projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-metadata-modal/documenten-api-metadata-modal.component.ts

```diff
@@ -579,27 +557,53 @@ export class DocumentenApiMetadataModalComponent implements OnInit, OnChanges, O
     this._subscriptions.add(
       combineLatest([this.file$, this.userEmail$])
         .pipe(
+          take(1),
```

Comment:
What's the point of adding a take(1) of observable to subscriptions? Are you sure adding this take(1) does not break previous functionality?

Reply:
I've removed it